components/TopBanner.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div
+    v-if="showBanner"
+    class="fixed w-screen h-[80px] md:h-[150px] sm:h-[200px] top-[0px] z-40 bg-black-950 flex md:flex-col justify-between items-center gap-md py-md px-lg md:px-md"
+  >
+    <div
+      class="grid grid-rows-[max-content_max-content] items-center gap-xs md:w-full md:pr-lg"
+    >
+      <p class="title text-white-50">{{ $t('banner.title') }}</p>
+      <p class="subtitle text-white-200">
+        {{ $t('banner.description') }}
+      </p>
+    </div>
+    <div
+      class="flex gap-xl md:gap-[0px] md:w-full md:justify-between items-center mr-lg md:mr-[0px]"
+    >
+      <div class="grid">
+        <div class="grid grid-cols-4 gap-xs justify-items-center items-center">
+          <p class="countdown-number">
+            {{ getFormatedCount(days) }}
+          </p>
+          <p class="countdown-number">
+            {{ getFormatedCount(hoursFormated) }}
+          </p>
+          <p class="countdown-number">
+            {{ getFormatedCount(minutesFormated) }}
+          </p>
+          <p class="countdown-number">
+            {{ getFormatedCount(secondsFormated) }}
+          </p>
+          <p class="countdown-label">Days</p>
+          <p class="countdown-label">Hrs</p>
+          <p class="countdown-label">Mins</p>
+          <p class="countdown-label">Secs</p>
+        </div>
+      </div>
+      <WButton :type="ButtonType.dark">
+        <a href="#" target="_blank">{{ $t('banner.action') }}</a>
+      </WButton>
+    </div>
+  </div>
+  <CloseNav
```

Comment:
Can we add a bit more space to the right of the close button and also making it slightly bigger? 
![image](https://github.com/user-attachments/assets/157e3b1c-03b5-49a3-a220-2f7b093519d5)